layouts/partials/fragments/items.html

```diff
@@ -57,14 +62,14 @@ <h5>
                 {{- printf " text-%s" "light" -}}
               {{- end -}}
             ">
-              {{ if .link }}
-                <a href="{{ .link | relLangURL }}" class="
+              {{ if $items.link }}
+                <a href="{{ $items.link | relLangURL }}" class="
                   {{- if eq $bg "primary" -}}
                     {{- printf " text-%s" "dark" -}}
                   {{- end -}}
-                ">{{ .name | markdownify }}</a>
+                ">{{ $items.name | markdownify }}</a>
               {{ else }}
-                {{- .name | markdownify -}}
+                {{- $items.name | markdownify -}}
               {{ end }}
             </h4>
             <p class="
```

Comment:
Using <div> instead of <p> solves the font color issue. The markdown will add it's own <p> and the font won't take effect. (at least that is my understanding).

Not sure, if it might be better in the long term to move more of the <p> usage to <div> as we allow markdown in most of these.

Reply:
Yeah i think that'll be better so this doesn't happen again.